plugins/scram.c

```diff
@@ -1538,6 +1537,10 @@ static int scram_server_mech_step(void *conn_context,
 	return SASL_BADPROT;
     }
 
+    scram_sasl_mech = (EVP_MD_size(text->md) == 32)
+        ? "SCRAM-SHA-256"
```

Comment:
I think ? should be on the previous line

Reply:
OK, I will revert to the original look. It is probably matter of personal preference.